bin/package/installation/post-install.sh

```diff
@@ -110,14 +110,18 @@ ensure_paths
 if [[ ! -f $DAEMON_DEFAULT ]]; then
     cp $OS_DIR_INSTALLATION/default $DAEMON_DEFAULT
 else
-    # TODO remove this hack when all nodes updated to both services.
+    # TODO remove these hacks when all nodes updates
     sed -i -e 's/SERVICE_OPTS="openvpn"/SERVICE_OPTS="openvpn,wireguard"/g' /etc/default/mysterium-node
     sed -i -e 's/--tequilapi.address=0.0.0.0/--tequilapi.address=127.0.0.1/g' /etc/default/mysterium-node
     # Append script-dir for existing installations only if missing
     sed -i -e '/^\(SCRIPT_DIR=\).*/{s//\1\"--script-dir=\/etc\/mysterium-node\"/;:a;n;ba;q}' \
            -e '$aSCRIPT_DIR="--script-dir=/etc/mysterium-node"' /etc/default/mysterium-node
 fi
 
+# TODO: remove me when all nodes updates
+printf "Ensuring that sudoers config is consistent...\n" \
+    && sed -i -e 's/\/sbin\/iptables/\/usr\/sbin\/iptables/g' /etc/sudoers.d/mysterium-node
```

Comment:
It's possible to use another delimiter for `sed` when you have a lot of `/` in the changes.
And It's better to include space in the expression to avoid recursion updates like `/usr/usr/sbin/iptables`
```suggestion
    && sed -i -e 's| /sbin/iptables| /usr/sbin/iptables|g' /etc/sudoers.d/mysterium-node
```

Reply:
I can match beggining of a line too.